Tests/PerceptionTests/PerceptionTests.swift

```diff
@@ -227,6 +255,19 @@ final class PerceptionTests: XCTestCase {
 
     self.render(FeatureView())
   }
+
+  func testRuntimeWarning_ActionClosure() {
+    struct FeatureView: View {
+      @State var model = Model()
+      var body: some View {
+        Text("Hi")
+          .onAppear { _ = self.model.count }
+      }
```

Comment:
I was able to get some basic test coverage on action closures by using `onAppear`. This test fails if the `!demangled.isActionClosure` check is removed from `perceptionCheck`.